generated/nifgen/session.py

```diff
@@ -3255,22 +3369,24 @@ def _error_message(self, error_code):
         return error_message_ctype.value.decode(self._encoding)
 
 
-class _RepeatedCapability(_SessionBase):
-    '''Allows for setting/getting properties and calling methods for specific repeated capabilities (such as channels) on your session.'''
-
-    def __init__(self, vi, repeated_capability):
-        super(_RepeatedCapability, self).__init__(repeated_capability)
-        self._vi = vi
-        self._is_frozen = True
-
-
 class Session(_SessionBase):
     '''An NI-FGEN session to a National Instruments Signal Generator.'''
 
     def __init__(self, resource_name, reset_device=False, option_string=""):
         super(Session, self).__init__(repeated_capability='')
+        self._library = library_singleton.get()
+        self._encoding = 'windows-1251'
         self._vi = 0  # This must be set before calling _initialize_with_channels().
         self._vi = self._initialize_with_channels(resource_name, reset_device, option_string)
+        self.channels = _Channels(self._vi, self._library, self._encoding)
```

Comment:
add a comment before you unroll this

`# Instantiate repeated capabilities`

Although if you go with the single class approach and the name is self-explanatory then the comment may become unnecessary.

Reply:
Added this plus some more comments in \_\_init\_\_